main_test.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import "testing"
+
+func TestLongAdd(t *testing.T) {
+	for _, tc := range []struct {
+		testn int
```

Comment:
Can you use meaningful test description instead, e.g. "zeros", "long number", "order overflow"?